dmoj/executors/base_executor.py

```diff
@@ -346,6 +410,7 @@ def get_compile_process(self):
                   'preexec_fn': self.create_executable_limits(), 'time_limit': self.compiler_time_limit}
         kwargs.update(self.get_compile_popen_kwargs())
 
+        log.debug("Compiling: `%s`", ' '.join(self.get_compile_args()))
```

Comment:
Do you actually want this?

Reply:
It's added as a debug-level call, and only the testsuite runs with logging configured for debug. I figured it would be a useful addition.